Cargo.toml

```
@@ -21,7 +21,7 @@ members = [
 resolver = "3"
 
 [workspace.package]
-version = "24.1.0"
+version = "24.2.0"
```

```suggestion
version = "24.1.1"
```

Let's follow semver. The difference between `24.1.0` and now is a bug fix.